tensorflow/core/kernels/whole_file_read_ops.cc

```diff
@@ -119,4 +127,31 @@ class ReadFileOp : public OpKernel {
 
 REGISTER_KERNEL_BUILDER(Name("ReadFile").Device(DEVICE_CPU), ReadFileOp);
 
+class WriteFileOp : public OpKernel {
+  public:
+    using OpKernel::OpKernel;
```

Comment:
If I remove this line compilation fails with "no matching constructor " error message.
`error: no matching constructor for initialization of 'tensorflow::WriteFileOp'
 note: candidate constructor (the implicit copy constructor) not viable: no known conversion from '::tensorflow::OpKernelConstruction *' to 'const tensorflow::WriteFileOp' for 1st argument
class WriteFileOp : public OpKernel {
`